protected/humhub/migrations/m240807_230603_add_permissions_to_existing_user_groups.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use humhub\modules\space\permissions\CreatePrivateSpace;
+use humhub\modules\space\permissions\CreatePublicSpace;
+use humhub\modules\user\models\Group;
+use yii\db\Migration;
+
+/**
+ * Class m240807_230603_add_permissions_to_existing_user_groups
+ */
+class m240807_230603_add_permissions_to_existing_user_groups extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        if (Yii::$app->isInstalled()) {
+            $groups = Group::find();
+
+            /** @var Group $group */
+            foreach ($groups->each() as $group) {
```

Comment:
@gevorgmansuryan Do you see a chance to handle this by raw db queries? 

For migrations, I would use as little business logic as possible, as this could change in the future and might have to be migrated as well.

Reply:
@luke- got it. Will be done